hw/mcu/dialog/da1469x/src/hal_gpio.c

```diff
@@ -491,6 +491,24 @@ mcu_gpio_set_pin_function(int pin, int mode, mcu_gpio_func func)
     __HAL_ENABLE_INTERRUPTS(primask);
 }
 
+void
+mcu_gpio_set_pin_function_raw(int pin, uint32_t raw_mode)
```

Comment:
Once this function is present maybe `mcu_gpio_set_pin_function()` (previous one) could just call this one to have just one way of dealing with interrupts and latching in case some modification is needed in the future.

Reply:
Good insight; thanks! Done.